Traceback (most recent call last):
  File "parser.py", line 50, in <module>
    ast = parser.parse(text, lexer,debug=0)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 331, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 1181, in parseopt_notrack
    tok = call_errorfunc(self.errorfunc, errtoken, self)
  File "/usr/local/lib/python2.7/dist-packages/ply/yacc.py", line 193, in call_errorfunc
    r = errorfunc(token)
  File "/home/nico/Desktop/tLeng2016(posta)/tleng2016/parser_rules.py", line 588, in p_error
    raise Exception(message)
Exception: [Syntax error]
type:[
value:[
line:1
position:44
